### PR TITLE
Completed the calendar task

### PR DESCRIPTION
**Pull request by Shevtsooov**
- [DEMO LINK](https://Shevtsooov.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,68 @@
...
+
+      <article class="calendar__day"></article>
+
+      <article class="calendar__day"></article>
+
+      <article class="calendar__day"></article>
+
+      <article class="calendar__day"></article>
+
+      <article class="calendar__day"></article>
```
Comment (snuber-w) [inline src/index.html]: I think that due to the fact that the rows are absolutely identical, there is no need to separate them. It does not improve readability ☘️

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @include cell-dimensions($day-width);
+    border: 1px solid black;
+    transition-duration: 0.5s;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+  }
```
Comment (snuber-w) [inline src/styles/blocks/calendar.scss]: check checklist #4 🍀

**Review by snuber-w — APPROVED** [review]
great 🔥

File: src/styles/blocks/calendar.scss (hunk 2 above)
Comment (Shevtsooov) [reply src/styles/blocks/calendar.scss]: I see what you mean, fixing it now.

Comment (Shevtsooov) [reply src/styles/blocks/calendar.scss]: @snuber-w Thanks! Fixed!

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    @include cell-dimensions($day-width);
+    border: 1px solid black;
+    transition-duration: 0.5s;
```
Comment (silchyn) [inline src/styles/blocks/calendar.scss]: you should specify properties for which you set transition

**Review by silchyn — APPROVED** [review]

File: src/styles/blocks/calendar.scss (hunk 3 above)
Comment (Shevtsooov) [reply src/styles/blocks/calendar.scss]: @silchyn Will do! Thanks!

Comment (Shevtsooov) [reply src/styles/blocks/calendar.scss]: @silchyn Done!

**Pull request closed without merging**